docs/antctl.md

```diff
@@ -422,6 +418,42 @@ results:
     action: Delivered
 ```
 
+To start a live-traffic traceflow, add `--live-traffic` (or `-L`) flag. Add
```

Comment:
add *the* ... flag

same in other places

Reply:
Ok.